activesupport/lib/active_support/cache/file_store.rb

```diff
@@ -43,14 +43,33 @@ def cleanup(options = nil)
         end
       end
 
-      # Increments an already existing integer value that is stored in the cache.
-      # If the key is not found nothing is done.
+      # Increment a cached integer value in the store. Returns the updated value.
+      #
+      # If the key is unset, it will be set to +amount+:
+      #
+      #   cache.increment("foo") # => 1
+      #   cache.increment("bar", 100) # => 100
+      #
+      # To set a specific value, call #write:
+      #
+      #   cache.write("baz", 5)
```

Comment:
I think this should be a raw write, right?
```suggestion
      #   cache.write("baz", 5, raw: true)
```

Reply:
A quick test using a file_store with default options is giving me this: 🤔 

```ruby
[24] pry> file_store.write("beebop", 2)
Cache write: test.txt/26E/530/beebop ({:compress=>true, :compress_threshold=>1024})
=> true
[25] pry)> file_store.increment("beebop")
Cache read: test.txt/26E/530/beebop ({:compress=>true, :compress_threshold=>1024})
Cache write: test.txt/26E/530/beebop ({:compress=>true, :compress_threshold=>1024})
=> 3
```

Reply:
Ah my bad, I missed that it was the `file_store`, that one doesn't need `raw` values for increment, so I guess it's fine.